prime-router/src/main/kotlin/Translator.kt

```diff
@@ -86,7 +86,7 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
             JurisdictionalFilters.defaultQualityFilters[receiver.topic] != null ->
                 JurisdictionalFilters.defaultQualityFilters[receiver.topic]!!
             else -> {
-                logger.info("No default quality filter found for topic ${receiver.topic}. Not doing qual filtering")
+                logger.info("No default qualityFilter found for topic ${receiver.topic}. Not doing qual filtering")
```

Comment:
A Lame Jim Hack:   I use the term qualityFilter in the log messages, so I can search for that term in the Azure logs easily.